packages/react/src/components/Tabs/Tab.tsx

```diff
@@ -0,0 +1,49 @@
+import React, { useRef, useEffect } from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+
+interface TabProps {
+  value: number;
+  index: number;
+  children?: React.ReactNode;
+}
+
+const Tab = ({ value, index, children }: TabProps) => {
+  const tabRef = useRef<HTMLLIElement>(null);
+  const tabIndex = index === value ? 0 : -1;
+  const selected = tabIndex === 0;
+
+  useEffect(() => {
```

Comment:
sorry, I should've been more specific when I made this suggestion but this useEffect runs on mount. This means that anytime anyone uses tabs focus will be stolen and set on the currently active tab. We only want this effect to run _after_ mounting. In a different project, I've setup a custom hook like this:

```ts
import React from 'react';

/**
 * Hook to be used similarly to the React.Component#componentDidMount.
 * Executes the provided `effect` when `dependencies` change but does not
 * execute the effect initially (on mount) - only on update.
 *
 * @param effect {Function} function to be executed when dependencies update
 * @param dependencies {Any} any valid dependency argument to React.useEffect
 */
const useDidUpdate = (effect: () => void, dependencies: unknown[]): void => {
  const mounted = React.useRef(false);

  React.useEffect(() => {
    if (!mounted.current) {
      mounted.current = true;
      return;
    }

    effect();
  }, dependencies);
};

export default useDidUpdate;

```

Reply:
What do you mean by `anytime anyone uses tabs focus will be stolen and set on the currently active tab`?

Reply:
I see. The 'Tab 1' is automatically focused on page loads.